app/forms/flood_risk_engine/steps/local_authority_form.rb

```diff
@@ -0,0 +1,47 @@
+module FloodRiskEngine
+  module Steps
+    class LocalAuthorityForm < FloodRiskEngine::Steps::BaseForm
+
+      def self.factory(enrollment)
+        organisation = enrollment.organisation || Organisation.new
+        new(organisation, enrollment)
+      end
+
+      def self.locale_key
+        "flood_risk_engine.enrollments.steps.local_authority"
+      end
+
+      def self.name_max_length
+        200
+      end
+
+      validation :name_present? do
+        validates :name, presence: {
+          message: I18n.t("#{LocalAuthorityForm.locale_key}.errors.name.blank")
+        }
+      end
+
+      include ActiveModel::Validations
+      validation :email_format, if: :name_present? do
```

Comment:
[allow_nil](http://guides.rubyonrails.org/active_record_validations.html#allow-nil) (or maybe [allow_blank](http://guides.rubyonrails.org/active_record_validations.html#allow-blank) better?) I didn't know about reform [validation groups](http://trailblazer.to/gems/reform/validation.html#validation-group) but the cynic in may says other devs are likely to think its a rails thing and scratch their heads a bit - I'd rather lean on pure Rails validations if poss without introducing too much reform-specifc magic :tophat: